app/controllers/application_controller.rb

```diff
@@ -1891,6 +1891,10 @@ def controller_for_common_methods
       "generic_object"
     when "ansible_playbook", "workflow"
       "embedded_configuration_script_payload"
+    # when "workflow_credential", "ansible_credential"
+    #  "embedded_automation_manager_credentials"
```

Comment:
https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_product_features.yml#L5239-L5242 would need to be changed to `embedded_automation_manager_credentials` first. I know you said in the past that this would require a data migration.

Reply:
I meant should they be deleted or have a `TODO` with a description, right now it looks like some debug code left in